ctry.py

```diff
@@ -111,27 +111,48 @@ def do_letter(self, l):
     def try1(self):
         if len(self.answer) == 0:
             self.message = 'please type in a country'
-            return
+            return -1
         l = self.answer[:1]
         ind = ord(l) - 65
         answer_fix = fix(self.answer)
-        ans = self.check(answer_fix)
-        if ans is None:
+        value, ans = self.check(answer_fix)
+        if ans is None or value == -1:
             self.message = 'sorry, ' + self.answer + ' is not in my list'
-            return
+            return -1
+        if value == 0:
+            self.message = 'Did you mean ' + ans + '? (y/n)'
+            return 0
         if g.answers[ind] != '':
             g.answers[ind] = ''
             self.redraw()
         g.answers[ind] = ans
+        # print(g.answers)

```

Comment:
You forgot to remove a debug statement perhaps. You can use `git add -p` to add chunks of lines, rather than files together